vllm/executor/multiproc_worker_utils.py

```diff
@@ -274,7 +274,17 @@ def write_with_prefix(s: str):
     file.write = write_with_prefix  # type: ignore[method-assign]
 
 
+def _check_multiproc_method():
+    if (cuda_is_initialized()
+            and os.environ.get("VLLM_WORKER_MULTIPROC_METHOD") != "spawn"):
+        logger.warning("CUDA was previously initialized. We must use "
```

Comment:
I think this should link back to the doc telling people to guard the import with `if __name__ == "__main__"`

Reply:
Done -- added a link to the appropriate section of the debugging tips page